website/docs/reference/warehouse-profiles/firebolt-profile.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Firebolt Profile"
+---
+
+
+:::info Community plugin
```

Comment:
This should be removed since this is a vendor supported adapter.

Reply:
```suggestion
```